main.nf

```diff
@@ -47,8 +47,8 @@ workflow {
         params.seqera_api_endpoint,
         params.skip_run_gantt,
         params.skip_multiqc,
-        params.java_truststore_path,
-        params.java_truststore_password
+        params.java_truststore_path ?: '',
+        params.java_truststore_password ?: ''
```

Comment:
Groovy handles empty string as null, therefore this should not be needed provided the downstream check is checking for emptiness instead of `null`. See https://groovy-lang.org/semantics.html#_strings

Reply:
> I tried running this on my QL2 test-runs by just switching out the revision. I guess the https://cloud.seqera.io/orgs/scidev/workspaces/testing/watch/3qH91IevSxwp0Z Is setting the param java_truststore_path mandatory?

Yep, this one I will remove but I did have to push another commit https://github.com/seqeralabs/nf-aggregate/pull/49/commits/3d05070a99daf520bf5208a80e078598fb747601 because to set the value as an empty string if `null` to deal with the run failure above.

Reply:
Done! https://github.com/seqeralabs/nf-aggregate/pull/49/commits/8821349cf36535319925adc7b05d025904627544